src/node_file.h

```diff
@@ -404,19 +410,19 @@ class FileHandle : public AsyncWrap, public StreamBase {
   FileHandle& operator=(const FileHandle&&) = delete;
 
  private:
+  FileHandle(Environment* env, v8::Local<v8::Object> obj, int fd);
+
   // Synchronous close that emits a warning
   void Close();
   void AfterClose();
 
   class CloseReq : public ReqWrap<uv_fs_t> {
    public:
     CloseReq(Environment* env,
```

Comment:
any reason why `CloseReq` does not follow the suit - i.e., a `private` constructor and a `static` factory?

Reply:
Just that it’s used at only one call site rather than multiple ones. I can add/use a factory constructor if you like?

Reply:
ok, I thought so. I think it is fine; doesn't look like anything beneficial of doing so.